packages/celotool/src/e2e-tests/utils.ts

```diff
@@ -339,3 +284,111 @@ export function getContext(gethConfig: GethRunConfig, verbose: boolean = verbose
     hooks: { before, after, restart, gethBinaryPath },
   }
 }
+
+// Modifies gethConfig to properly configure proxies and proxied instances
+export function configureInstances(gethConfig: GethRunConfig, bootnodeEnode?: string) {
```

Comment:
A less generic name would be helpful.
```suggestion
export function setProxyConfigurations(gethConfig: GethRunConfig, bootnodeEnode?: string) {
```